src/Testcontainers.LowkeyVault/LowkeyVaultBuilder.cs

```diff
@@ -0,0 +1,334 @@
+            .WithResourceMapping(new FileInfo(keyStoreFilePath), new FileInfo("/config/cert.store"));
+    }
+
+    /// <summary>
+    /// Enable Or Disable Debug.
+    /// </summary>
+    /// <param name="debug">The flag to enable or disable debug.</param>
+    /// <returns>A configured instance of <see cref="LowkeyVaultBuilder" />.</returns>
+    public LowkeyVaultBuilder WithDebug(bool debug)
+    {
+        return WithEnvironment(LowKeyVaultEnvVarKey, AddOrAppend($"--LOWKEY_DEBUG_REQUEST_LOG={debug}"));
+    }
+
+    /// <summary>
+    /// Enable Or Disable Relaxed Ports.
+    /// </summary>
+    /// <param name="relaxedPorts">The flag to enable or disable relaxed ports.</param>
+    /// <returns>A configured instance of <see cref="LowkeyVaultBuilder" />.</returns>
+    public LowkeyVaultBuilder WithRelaxedPorts(bool relaxedPorts)
+    {
+        return WithEnvironment(LowKeyVaultEnvVarKey, AddOrAppend($"--LOWKEY_VAULT_RELAXED_PORTS={relaxedPorts}"));
+    }
+
+    /// <summary>
+    /// Sets The host used to replace host placeholder in import template.
+    /// </summary>
+    /// <param name="logicalHost">The logical host.</param>
+    /// <returns>A configured instance of <see cref="LowkeyVaultBuilder" />.</returns>
+    public LowkeyVaultBuilder WithLogicalHost(string logicalHost)
+    {
+        return WithEnvironment(LowKeyVaultEnvVarKey, AddOrAppend($"--LOWKEY_IMPORT_TEMPLATE_HOST={logicalHost}"));
+    }
+
+    /// <summary>
+    /// Sets The port used to replace host placeholder in import template.
+    /// </summary>
+    /// <param name="logicalPort">The logical port.</param>
+    /// <returns>A configured instance of <see cref="LowkeyVaultBuilder" />.</returns>
+    public LowkeyVaultBuilder WithLogicalPort(ushort logicalPort)
+    {
+        return WithEnvironment(LowKeyVaultEnvVarKey, AddOrAppend($"--LOWKEY_IMPORT_TEMPLATE_PORT={logicalPort}"));
+    }
+
+
+    /// <summary>
+    /// Sets Additional Arguments.
+    /// </summary>
+    /// <param name="debug">The additional arguments.</param>
+    /// <returns>A configured instance of <see cref="LowkeyVaultBuilder" />.</returns>
+    public LowkeyVaultBuilder WithAdditionalArguments(List<string> additionalArguments)
+    {
+        return Merge(DockerResourceConfiguration, new LowkeyVaultConfiguration(additionalArguments: additionalArguments))
+            .WithEnvironment(LowKeyVaultEnvVarKey, AddOrAppend(string.Join(" ", additionalArguments)));
+    }
+
+    /// <inheritdoc />
+    public override LowkeyVaultContainer Build()
+    {
+        Validate();
+
+        var waitStrategy = Wait.ForUnixContainer().UntilMessageIsLogged("(?s).*Started LowkeyVaultApp.*$");
+
+        var lowkeyVaultBuilder = DockerResourceConfiguration.WaitStrategies.Count() > 1 ? this : WithWaitStrategy(waitStrategy);
+
+        return new LowkeyVaultContainer(lowkeyVaultBuilder.DockerResourceConfiguration);
```

Comment:
```suggestion
        Validate();
        return new LowkeyVaultContainer(DockerResourceConfiguration);
```

Reply:
From what I see with your suggested amendment, you are removing the custom wait strategy, any reason for that?

Reply:
updated in the latest commit, thanks.